ReviewMonitor/AppHelper.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  AppHelper.swift
+//  ReviewMonitor
+//
+//  Created by Tayal, Rishabh on 1/3/18.
+//  Copyright © 2018 Tayal, Rishabh. All rights reserved.
+//
+
+import UIKit
+
+enum DateFormat: String {
+    case MMMddyyy = "MMM dd, yyyy"
```

Comment:
I believe that enum cases should be lowercase. [Proposal SE-0006](https://github.com/apple/swift-evolution/blob/master/proposals/0006-apply-api-guidelines-to-the-standard-library.md)

Reply:
You are right but I thought we should keep it the same as the string value otherwise we won't be able to distinguish between `mm` and `MM`. What would be the `case` in case of `MMddyyyy` and `mmddyyyy`?